Sources/StreamChatSwiftUI/ChatChannel/MessageList/VideoAttachmentView.swift

```diff
@@ -86,10 +88,12 @@ public struct VideoAttachmentView: View {
                 Button {
                     fullScreenShown = true
                 } label: {
-                    Image(systemName: "play.fill")
-                        .font(.system(size: 30))
+                    Image(uiImage: images.playFilled)
+                        .renderingMode(.template)
+                        .resizable()
+                        .scaledToFit()
+                        .frame(width: 24)
                         .foregroundColor(.white)
-                        .padding(.all, 32)
```

Comment:
This was used to increase the tappable area of the play control. Is there any specific reason why it was removed?

Reply:
I was wondering if it fills any purpose but couldn't find it. That's why I removed it, which was a mistake. Reverted the change.